server/plugin/command.go

```diff
@@ -572,6 +572,14 @@ func getAutocompleteData(config *Configuration) *model.AutocompleteData {
 	settings.AddStaticListArgument("", true, value)
 	github.AddCommand(settings)
 
+	issue := model.NewAutocompleteData("issue", "[command]", "Available commands: create")
+
+	issueCreate := model.NewAutocompleteData("create", "[description]", "Open a dialog to create a new issue in Github, using the description if provided")
+	issueCreate.AddTextArgument("Description for the Github issue", "[description]", "")
```

Comment:
Given that the title is the only required parameter, I'm wondering if the user should be able to input that instead of the description.

Reply:
I think it probably does make more sense for this to be title, you're more likely to want to note down something short in Mattermost and fill it in properly in the Github UI.

The reason I went for description here is that is how the Jira command is implemented, but considering these are separate commands, I don't think we have to follow that convention.

Reply:
Made this change in the latest push.